package.json

```diff
@@ -145,11 +145,19 @@
       ]
     }
   },
+  "sideEffects": false,
   "exports": {
     ".": {
-      "import": "./dist/index.mjs",
-      "require": "./dist/index.js",
-      "default": "./dist/index.js",
+      "import": "./dist/index.min.mjs",
```

Comment:
should not be minified so it seems

Reply:
If there is something wrong with this change open a visible issue, comments on old PR have basically no visibility unless i m combing the emails one by one.

Reply:
@asturur  The non-minified build is basically never picked up. This should have been done like React, where it picks the correct one based on the ENV:

![Screenshot 2024-04-18 at 14 03 02](https://github.com/fabricjs/fabric.js/assets/10067273/9bba0174-0bd2-4c6e-8fab-cd5559c35460)

Forcing the minified build during development is just more painful for debugging. You can also reuse the same strategy to strip fabric's logs for the production build.

Reply:
important is that production has the minified build, if you want top open a change to surface the non minified build in dev you can do it.
Surfacing non minified code in production seems as bad as minified in dev.

Regarding the affected test, what snapshot is it about?

Reply:
Classes get renamed in the minified build so that affects Jest snapshots that included those classes, like Point. 

I’ll open a PR for the production build then.

Reply:
Anyway if the non production build ends in production I wonder if it means that you have an issue with the app bundler, as I think node modules are typically distributed unminified, then the bundler can transpile and minify them in production. For instance one may want to ship as ES5 and there are as well different ways to minify.
But since React adopts the minified build directly on production I guess that’s the standard.

Reply:
lodash, redux, ships unminified, lottie player ships unminified. Those are random picks i made.
Important is that there is something for the browser inclusion that is still minified.

Reply:
On a look at `rollup.config.mjs`, I don't know exactly who the fabric build system works so I'll leave the PR to you guys if you want to do it. Otherwise no problem, we can wait if someone else has issues. Internally at my workplace we've just patched `package.json` to continue using the unminified build.

Reply:
But do sourcemaps works for you or not?

Reply:
I opened this PR because my VITE build was picking up the unminified code, so vite wasn't minifying anything for me